k2/csrc/rnnt_decode.cu

```diff
@@ -159,8 +159,8 @@ void RnntDecodingStreams::GetContexts(RaggedShape *shape,
         int64_t state_value = states_values_data[state_idx01x],
                 context_state = state_value / num_graph_states,
                 exp = decoder_history_len - col,
-                state = context_state % (int64_t)powf(vocab_size, exp);
-        state = state / (int64_t)powf(vocab_size, exp - 1);
+                state = context_state % (int64_t)pow(vocab_size, exp);
```

Comment:
You can use
```cpp
#ifndef _MSC_VER
#define K2_POW pow
#else
#define K2_POW powf
#endif

Reply:
Yes, I agree. But I am not sure whether this bug would happen on windows platform.

Reply:
One thing you can do is to add a CI test in icefall using fast beam search for windows.

- https://k2-fsa.org/nightly/whl/k2-1.14.dev20220409+cpu.torch1.10.0-cp38-cp38-win_amd64.whl
- https://github.com/k2-fsa/icefall/blob/master/.github/workflows/run-librispeech-2022-03-12.yml

Reply:
It needs that number to be exact, not approximate.  I think it would make more sense to code an int64 version of 'pow', rather than using floating-point versions.